policy/delegations.md

```diff
@@ -0,0 +1,222 @@
+# 1. Juno Delegation Program
+
+## 1.1 Juno Delegations SubDAO Inception
+
+The Juno Delegation Program (JDP) is part of the Juno Delegations SubDAO, which was established through Juno governance according to [proposal #50](https://www.mintscan.io/juno/proposals/50 "Juno Network Governance Proposal no. 50 on Mintscan").
+
+## 1.2 Why do we have a delegation program?
+
+The three core components of any distributed proof of stake blockchain are delegators, validators and the founding team. As the blockchain matures, additional roles are required to assist with community engagement and support to build on foundations set from genesis. Typical members/roles added include developers, marketing managers, and community managers. As Juno is a DAO-based blockchain, managing delegations to validators from the Juno Development Fund has been (primarily) moved from C-Root to the Juno Delegation SubDAO.
```

Comment:
"moved from C-Root" 

"moved from the founding team (aka "C-Root")

Just for future references incase there is a name change etc.

Reply:
Update has been made